src/main/java/io/jenkins/plugins/gitlabbranchsource/GitLabHookCreator.java

```diff
@@ -84,14 +84,15 @@ public static void register(GitLabSCMSource source,
                 return;
         }
         String hookUrl = getHookUrl(server, true);
+        String secretToken = server.getSecretToken().getPlainText();
```

Comment:
If secretToken is null it leads to Exception https://pastebin.com/ptxv5DLz

Reply:
I created a PR against this. #105
Please feel free to help me to review it.